pygfx/objects/_lights.py

```diff
@@ -643,25 +643,13 @@ def _update_matrix(self, light: Light) -> None:
             camera.update_projection_matrix()
 
         for i in range(6):
-            camera.position.set_from_matrix_position(light.matrix_world)
+            camera.look_at(directions[i])
```

Comment:
Maybe add a comment to explain why the up-vector does not matter.

Reply:
Done.